src/jira/client/jira-client.ts

```diff
@@ -286,7 +294,8 @@ export const getJiraClient = async (
 				const payload = {
 					deployments: data.deployments,
 					properties: {
-						gitHubInstallationId
+						gitHubInstallationId,
+						repositoryId
```

Comment:
Adding the `repositoryId` as a property when sending a deployment, so we can delete by that repositoryId later.